dist/api_data.py

```diff
@@ -803,7 +803,12 @@ def __cmp__(self, other):
         type='boolean'),
 ]),
 
-'WT_SESSION.commit_transaction' : Method([]),
+'WT_SESSION.commit_transaction' : Method([
+    Config('sync', 'off', r'''
+        override whether to sync log records when the transaction commits,
+        inherited from ::wiredtiger_open \c transaction_sync''',
+        choices=['background', 'off', 'on']),
+]),
```

Comment:
This suggests the default is "off".  In fact, I think the default is to inherit on/off from `wiredtiger_open`.  I would leave the default as an empty string, which might mean adding the empty string to the list of choices.